Optik/Classes/AlbumViewController.swift

```diff
@@ -77,7 +77,8 @@ internal final class AlbumViewController: UIViewController {
          initialImageDisplayIndex: Int,
          activityIndicatorColor: UIColor?,
          dismissButtonImage: UIImage?,
-         dismissButtonPosition: DismissButtonPosition) {
+         dismissButtonPosition: DismissButtonPosition,
+         likeButtonPosition: LikeButtonPosition) {
```

Comment:
Could we get rid of the concept of like button all together and call it actionButton? And both the image and associated action would come from the outside. I already mentioned that in my previous comment but looks like it's still there. Do you have any issue with that idea?